app/controllers/admin_general_controller.rb

```diff
@@ -8,14 +8,22 @@ class AdminGeneralController < AdminController
 
   def index
     # Tasks to do
-    @requires_admin_requests = InfoRequest.
-      find_in_state('requires_admin').
+    @requires_admin_requests =
+      InfoRequest.
+        find_in_state('requires_admin').
+        where(awaiting_description: false).
         not_embargoed
-    @error_message_requests = InfoRequest.
-      find_in_state('error_message').
+
+    @error_message_requests =
+      InfoRequest.
+        find_in_state('error_message').
+        where(awaiting_description: false).
         not_embargoed
-    @attention_requests = InfoRequest.
-      find_in_state('attention_requested').
+
+    @attention_requests =
+      InfoRequest.
+        find_in_state('attention_requested').
+        where(awaiting_description: false).
```

Comment:
I think there's some subtlety to address with the requires_admin/attention_requested cases.

Note to self: A request might be awaiting classification by the author because it's received an auto-response, but then a different user might report it because there's a different issue with it (offensive, or whatever). Does that retain the `awaiting_description: false` state?

Also check notes in https://github.com/mysociety/alaveteli/issues/852.